app/assets/html/survey_questions.html

```diff
@@ -1,6 +1,6 @@
 <div id="survey-questions-container">
     <div class="header">{{ currentTopic }}</div>
-    <p id="description">{{ pageDescription() }}</p>
+    <p id="description">{{ numQuestions }} questions.</p>
```

Comment:
If you want to change the description, change it in the pageDescription method of surveyQuestions.coffee. What you did here breaks state knowledge.